src/main.rs

```diff
@@ -446,6 +448,9 @@ fn error<T: std::fmt::Display>(msg: T, location: Location, file_db: &Files, colo
             // glados quotes
             "I'd just like to point out that you were given every opportunity to succeed",
             "Okay. Look. We both did a lot of things that you're going to regret. But I think we can put our differences behind us. To prevent segfaults. You monster.",
+            "Congratulations. Not on the code.",
+            "You know, if you'd done that to some other codebase, they might devote their existence to exacting revenge.",
+            "Not bad. I forgot how good you are at this. You should pace yourself, though. We have A LOT of code to compile.",
```

Comment:
Hmm, this seems a little odd for an error. I like the quote but I'm not sure it fits.

Reply:
Hm, okay, I'll leave it out for now and see if i can make it more fitting later